AccountService/src/User.java

```diff
@@ -1,30 +1,30 @@
 public class User {
-    private int customerId;
-    private String userName;
-    private String password;
-    private boolean admin;
-    public String getUserName() {
-        return userName;
-    }
-    public void setUserName(String newUsername){
-       this.userName = newUsername;
-    }
-    public int getCustomerId(){
-        return customerId;
-    }
-    public void setCustomerId(int newCustomerId){
-        this.customerId = newCustomerId;
-    }
-    public String getPassword() {
-        return password;
-    }
-    public void setPassword(String newPassword){
-        this.password = newPassword;
-    }
-    public boolean getAdmin(){
-        return admin;
-    }
-    public void setAdmin(boolean newAdmin){
-        this.admin = newAdmin;
-    }
+	private int customerId;
+	private String userName;
+	private String password;
```

Comment:
should we set these as `private final String customerId/userName/password`, since the `final` keyword will make sure any User object created will alway have these fields. This can avoid possible nullPointerException